input/examples/practitioner-darleen-mackay.xml

```diff
@@ -17,6 +19,18 @@
             <display value="Docker River Nephrology Clinic"/>
         </assigner>
```

Comment:
Move the employee number to the PractitionerRole since it is role dependent.

Reply:
Done